package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Could you rebase on master and update this file accordingly, I've updated the dependencies on master? Thanks

Reply:
Done.

Reply:
Done.